src/Apps/Order2/Routes/Details/Components/Order2DetailsFulfillmentInfo.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box, Spacer, Text } from "@artsy/palette"
+import type { Order2DetailsFulfillmentInfo_order$key } from "__generated__/Order2DetailsFulfillmentInfo_order.graphql"
+import type React from "react"
+import { useFragment } from "react-relay"
+import { graphql } from "relay-runtime"
+
+interface Props {
```

Comment:
@oxaudo - this is a super minor thing, but it is consistent with the rest of the app! @erikdstock has been fixing some of the inconsistencies [in here](https://github.com/artsy/force/pull/15635), if we can stick to the pattern 🙏